geoportal/tests/functional/conftest.py

```diff
@@ -25,19 +25,29 @@
 # of the authors and should not be interpreted as representing official policies,
 # either expressed or implied, of the FreeBSD Project.
 
+import logging
+from typing import TYPE_CHECKING, Any
 from unittest.mock import patch
 
+import pyramid.request
 import pytest
 import sqlalchemy.orm
 import transaction
 from c2c.template.config import config as configuration
 from pyramid.testing import DummyRequest
+from sqlalchemy.orm.session import Session, SessionTransaction
```

Comment:
For my understanding, why this is not under the `if TYPE_CHECKING:` ? 
It's always used as a type.
But maybe that doesn't really matter ?